pom.xml

```diff
@@ -70,6 +70,12 @@
       <artifactId>pusher-http-java</artifactId>
       <version>1.0.0</version>
     </dependency>
+    <dependency>
```

Comment:
It looks like this dep is already in master:
https://github.com/googleinterns/step105-2020/blob/master/pom.xml

Could you merge master into this branch to remove this diff?